srtcore/group.cpp

```diff
@@ -1732,11 +1837,11 @@ void CUDTGroup::fillGroupData(SRT_MSGCTRL&       w_out, // MSGCTRL to be written
     w_out.grpdata = grpdata;
 }
 
-struct FLookupSocketWithEvent
+struct FLookupSocketWithEvent_LOCKED
 {
     CUDTUnited* glob;
     int         evtype;
-    FLookupSocketWithEvent(CUDTUnited* g, int event_type)
+    FLookupSocketWithEvent_LOCKED(CUDTUnited* g, int event_type)
```

Comment:
1. Please follow some naming convention
2. What is expected to be locked?

Reply:
Actually both `CUDTUnited::m_GlobControlLock` and `CUDTGroup::m_GroupLock` should be locked. The special `_LOCKED` marker is added here because this is a functional so this is the only way how it can be eye-catching at the usage point.

Reply:
Where is the special comment?
```c++
// [[using locked(m_GlobControlLock)]]
```

Reply:
Appropriate markers added.